client/wfprev-war/src/main/angular/src/assets/data/appConfig.json

```diff
@@ -6,7 +6,7 @@
     "webade": {
         "oauth2Url": "#{WEBADE_OAUTH2_CHECK_AUTHORIZE_URL}#",
         "clientId": "WFNEWS-UI",
-        "authScopes": "WFNEWS.*",
+        "authScopes": "WFNEWS.* WFDM.*",
```

Comment:
We should be replacing WFNEWS.* stuff with WFPREV as soon as we can. The permissions exist through webade already